### PR TITLE
Added content photos

### PR DESCRIPTION
**Pull request by brodi-xx**
Product photos and logo

### CODE REVIEW

File: front-end/images/jam.webp
Comment (Jstanion) [inline front-end/images/jam.webp:1]: image file isn't showing, check that the URL is accurate

File: front-end/images/rawHoney.avif
Comment (Jstanion) [inline front-end/images/rawHoney.avif:1]: image file isn't showing, check that the URL is accurate

**Review by Jstanion — CHANGES_REQUESTED** [review]
I love the "local farm" feel of these photos. Good find!

There are two image files that do not display anything - please check to confirm the URL is accurate for each.

**Review by brodi-xx — COMMENTED** [review]
I will change out the faulty images once I figure out how

**Comment by brodi-xx** [comment]
Faulty images, will resubmit with correct formats

**Pull request closed without merging**